applications/MeshingApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -154,6 +155,7 @@ void AddCustomUtilitiesToPython()
     ("MmgUtility3D", init<ModelPart&>())
     .def(init<ModelPart&, const std::string>())
     .def(init<ModelPart&, const std::string, const unsigned int>())
+    .def(init<ModelPart&, const std::string, const unsigned int, const std::string>())
```

Comment:
why don't you pass directly the enum in the constructor?
also wouldn't it start to make sense passing a "Parameters"?